catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/CoreAttributes.java

```diff
@@ -54,8 +54,8 @@
         descriptors.add(new AttributeDescriptorImpl(DESCRIPTION,
                 true /* indexed */,
                 true /* stored */,
-                true /* tokenized */,
```

Comment:
I think description should be tokenized, otherwise it will only hit on exact matches.